documentation-site/components/yard/overrides.tsx

```diff
@@ -66,58 +78,118 @@ const Overrides: React.FC<TOverridesProps> = ({
   };
 
+  const expanded = Object.keys(overrides.value ? overrides.value : {})
+    .map(key => {
+      const override = overrides.value[key];
+      if (override.active) {
+        return key;
+      } else {
+        return null;
+      }
+    })
+    .filter(val => !!val);
+
   return (
     <React.Fragment>
-      <Caption1
-        marginLeft="scale200"
-        marginRight="scale200"
-        marginBottom="scale400"
-      >
-        Additionally, you can fully customize any part of the {componentName}{' '}
-        component through the overrides prop (
-        <Link href="/guides/understanding-overrides">
-          <StyledLink href="/guides/understanding-overrides">
-            learn more
-          </StyledLink>
-        </Link>
-        ). Try to update different <b>style overrides</b> in the explorer
-        bellow:
-      </Caption1>
       <Accordion
-        initialState={{
-          expanded: overrides.value ? Object.keys(overrides.value) : [],
+        overrides={{
+          Root: {
+            style: {
+              marginLeft: isNested ? '8px' : '0px',
+              width: 'auto',
+            },
+          },
+          Header: {
+            style: {
+              paddingTop: '8px',
+              paddingBottom: '8px',
+              paddingLeft: '8px',
+              paddingRight: '8px',
+              fontSize: '16px',
+              borderBottomWidth: 0,
+            },
+          },
+          Content: {
+            style: {
+              backgroundColor: 'transparent',
+              paddingTop: 0,
+              paddingBottom: 0,
+              paddingLeft: '8px',
+              paddingRight: 0,
+              borderBottomWidth: 0,
+            },
+          },
         }}
+        expanded={expanded as string[]}
         onChange={handleChange}
         accordion={false}
       >
-        {Object.keys(overridesObj).map(overrideKey => (
-          <Panel
-            key={overrideKey}
-            title={overrideKey}
-            overrides={{
-              Content: {
-                style: {
-                  backgroundColor: 'transparent',
-                  paddingLeft: 0,
-                  paddingRight: 0,
-                },
-              },
-            }}
-          >
-            <Override
+        {Object.keys(overridesObj).map(overrideKey => {
+          const {nested} = overridesObj[overrideKey];
+          return (
+            <Panel
               key={overrideKey}
-              overrideKey={overrideKey}
-              overridesObj={overridesObj}
-              overrides={overrides}
-              componentConfig={componentConfig}
-              componentName={componentName}
-              set={set}
-            />
-          </Panel>
-        ))}
+              title={
+                <span>
+                  {overrideKey}
+                  {nested ? (
+                    <NestedTooltip
+                      name={componentName}
```

Comment:
There are some nested overrides that don't have the same key as the component name. For example there are multiple 'Select' components in the datepicker. May need to add some key value pair in the config file

https://github.com/uber/baseweb/blob/master/src/datepicker/types.js#L29

Reply:
for that case it should be enough to do

```
overrides: {
  custom: {
    names: [
      'Root',
      {...TagConfig, componentName: 'TagRenamed' }
```

if not, will come up with some different fix later